docs/en/use_case/OpenMLDB_Byzer_taxi.md

```diff
@@ -0,0 +1,275 @@
+# Build End-to-end Machine Learning Applications Based on SQL (OpenMLDB + Byzer)
+
+This tutorial will show you how to complete a machine learning workflow with the help of [OpenMLDB](https://github.com/4paradigm/OpenMLDB) and [Byzer](https://www.byzer.org/home).
+The OpenMLDB will compute the features of the real-time data in terms of the commands and data sent by Byzer and will return the dataset been processed to Byzer for later training and prediction.
```

Comment:
OpenMLDB will compute real-time features based on the data and queries from Byzer, and then return results to Byzer for subsequent model training and inference.